Dockerfile

```diff
@@ -10,6 +10,5 @@ COPY bmw_cd_exporter.py /usr/src/app
 COPY attributes.json /usr/src/app
 
 EXPOSE 9488
-ENV BMWCD_PORT=9488 BMWCD_INTERVAL=5 BMWCD_REGION=rest_of_world DEBUG=0
```

Comment:
Why is this removed please?

Reply:
i wanted to use docke-compose and use in the docker-compose.yml, environments variables.

Reply:
You can still use those.  These here are for defaults only.

Reply:
Revert to intial values.